doc/controls/DrawerFlyoutPresenter.md

```diff
@@ -15,26 +15,71 @@ xmlns:utu="using:Uno.Toolkit.UI"
        BasedOn="{StaticResource DrawerFlyoutPresenterStyle}"
        TargetType="FlyoutPresenter">
     <Setter Property="utu:DrawerFlyoutPresenter.OpenDirection" Value="Top" />
+    <Setter Property="utu:DrawerFlyoutPresenter.DrawerDepth" Value="0.66*" />
```

Comment:
`DrawerLength` maybe? Would be more in line with the naming of the `GridLength` type.

I don't mind Depth but just opening the discussion on Length instead

Reply:
I was debating myself on this too. I guess Length is probably better, and serves a better transition from DrawerDepth (which was an existing property of double), since both properties are not 1:1.